src/main/java/com/citihub/configr/exception/SchemaValidationException.java

```diff
@@ -6,9 +6,7 @@
 public class SchemaValidationException extends HttpClientErrorException {
 
   public SchemaValidationException(String validationError) {
-    super(HttpStatus.INTERNAL_SERVER_ERROR,
-        "Schema validation failed: "
-    		+ validationError);
+    super(HttpStatus.PRECONDITION_FAILED, "Schema validation failed: " + validationError);
```

Comment:
Actually a 422, Unprocessable Entity is more accurate here than precondition failed apparently. It's a WebDAV extension, not vanilla Http, but seems like a better choice #nicetohave

Reply:
Done